tracing-jersey/src/main/java/com/palantir/tracing/jersey/TraceEnrichingFilter.java

```diff
@@ -120,4 +120,13 @@ private static Observability getObservabilityFromHeader(ContainerRequestContext
             return "1".equals(header) ? Observability.SAMPLE : Observability.DO_NOT_SAMPLE;
         }
     }
+
+    private enum ContainerResponseContextTagRecorder implements TagRecorder<ContainerResponseContext> {
+        INSTANCE;
+
+        @Override
+        public <T> void record(TagAdapter<T> sink, T target, ContainerResponseContext data) {
+            sink.tag(target, "status", Integer.toString(data.getStatus()));
```

Comment:
Seeing the dialogue one made me think: would it not make sense to have a #tag method that takes some more common values too? like int/long/boolean? or just object?

Reply:
Hmm, I could go either way on this. I've repeated the Integer.toString piece a few times, for the same status code data. The current structure makes it obvious that everything is encoded into a string for the tracing system, where object may lead to confusion about the representation (how is ArrayList of [1,2,3] represented? not obvious that toString values aren't helpful in tracing, and may be more expensive than the tracing operation itself).

We have the flexibility to add those overloads later on using default methods if we need to.

Reply:
Well, deep down in the rabbithole, this just adds it to a conjure type, which eventually gets shipped as jackson... Anyway, probably not worth it, but it looked like it could be a useful extension.